julearn/scoring.py

```diff
@@ -26,6 +26,9 @@ def get_extended_scorer(estimator, score_name):
     scorer = check_scoring(estimator, scoring=score_name)
 
     def extended_scorer(estimator, X, y):
-        y_true = estimator.transform_target(X, y)
+        try:
+            y_true = estimator.transform_target(X, y)
+        except AttributeError:
```

Comment:
I personally do not agree to use exceptions to handle code flow. The correct way here will be to use `hasattr(estimator, 'transform_target')`